binary_transparency/firmware/docs/design/README.md

```diff
@@ -129,7 +129,55 @@ The design for the demo consists of a number of different entities which play
 the roles described in the claimant model above, these are shown in the
 following diagram:
 
-![overview diagram](./overview.svg)
+![ov](./ov.svg)
+<div style="display: none">
+<!--
+This is an embedded PlantUML config for the overview diagram.
+If you change it, be sure to update the generated diagram by running `plantuml -tsvg  README.md`!
+-->
+@startuml ov
```

Comment:
I think overview is a better name than ov. Can we go back to that?

Reply:
done